### ISSUE TITLE
Model train is not effective 

### ISSUE THREAD

**Issue by HarshkumarDegamadiya** [ShivamShrirao/diffusers#179]
### Describe the bug

When i trained model with my image the grid was showing worst resul and when i generated image , it is showing me my image only..
No matter what prompt i use , it is showing same potrait image

### System Info

Colab

**Comment by 2blackbar** [comment]
well, something major got changed on diffusers side, it broke entire dreambooth training from what ive seen. today, it was working fine 2 days ago

**Comment by ShivamShrirao** [comment]
Even with https://github.com/ShivamShrirao/diffusers/pull/178 ?

**Comment by HarshkumarDegamadiya** [comment]
yes even with the #178 .

**Comment by 2blackbar** [comment]
yes mine just crapped when trying to generate images at 500 steps , we have to wait for a fix, even my saved colab notebooks didnt help cause code is being downloaded from here anyway , thats why i dont like python, one dependency gets "update" and everything else craps down like a house of cards

**Comment by HarshkumarDegamadiya** [comment]
i think it is the error

/usr/local/lib/python3.8/dist-packages/bitsandbytes/cuda_setup/paths.py:105: UserWarning: /usr/lib64-nvidia did not contain libcudart.so as expected! Searching further paths...
  warn(
/usr/local/lib/python3.8/dist-packages/bitsandbytes/cuda_setup/paths.py:27: UserWarning: WARNING: The following directories listed in your path were found to be non-existent: {PosixPath('--listen_host=172.28.0.12 --target_host=172.28.0.12 --tunnel_background_save_url=https'), PosixPath('//colab.research.google.com/tun/m/cc48301118ce562b961b3c22d803539adc1e0c19/gpu-t4-s-2vzb2yx032dzy --tunnel_background_save_delay=10s --tunnel_periodic_background_save_frequency=30m0s --enable_output_coalescing=true --output_coalescing_required=true')}
  warn(
/usr/local/lib/python3.8/dist-packages/bitsandbytes/cuda_setup/paths.py:27: UserWarning: WARNING: The following directories listed in your path were found to be non-existent: {PosixPath('6000,"kernelManagerProxyHost"'), PosixPath('{"kernelManagerProxyPort"'), PosixPath('"172.28.0.12","jupyterArgs"'), PosixPath('["--ip=172.28.0.12","--transport=ipc"],"debugAdapterMultiplexerPath"'), PosixPath('true}'), PosixPath('"/usr/local/bin/dap_multiplexer","enableLsp"')}
  warn(
/usr/local/lib/python3.8/dist-packages/bitsandbytes/cuda_setup/paths.py:27: UserWarning: WARNING: The following directories listed in your path were found to be non-existent: {PosixPath('module'), PosixPath('//ipykernel.pylab.backend_inline')}
  warn(
/usr/local/lib/python3.8/dist-packages/bitsandbytes/cuda_setup/paths.py:27: UserWarning: WARNING: The following directories listed in your path were found to be non-existent: {PosixPath('/env/python')}
  warn(
/usr/local/lib/python3.8/dist-packages/bitsandbytes/cuda_setup/paths.py:27: UserWarning: WARNING: The following directories listed in your path were found to be non-existent: {PosixPath('/sys/fs/cgroup/memory.events /var/colab/cgroup/jupyter-children/memory.events')}

**Comment by tchesket** [comment]
Yep I am having the same issue. Was working fine yesterday but today the results are terrible. Getting the same CUDA_SETUP error, then it still trains but it isn't working properly.

**Comment by KitaharaMugiro** [comment]
The error message says "TypeError: unwrap_model() got an unexpected keyword argument 'keep_fp32_wrapper'"
So I removed keep_fp32_wrapper from unwrap_model and it worked. #181

**Comment by KitaharaMugiro** [comment]
I closed #181 because the issue is solved by updating 'accelerate' library

**Comment by maxdaneau** [comment]
I imagine this is related to why a derivative dreambooth notebook I'm using is failing. "train_dreambooth.py" is just not saving the final output directory for me anymore (I'm expecting a "0" folder and a "800" folder because I'm using 800 steps, but there's only a "0" directory after it's done running).

**Comment by enoreyes** [comment]
I am having the same issue, I'm able to train but the prompted outputs do not look any different from the un-prompted generation of the fine-tuned model.

**Comment by ShivamShrirao** [comment]
@enoreyes it's related to bitsandbytes. Uninstall your mistakes and install the correct version.

**Comment by enoreyes** [comment]
By "correct version", you mean 0.35.4 correct? I was able to get it working using that version.
